engine/src/main/scala/cromwell/webservice/routes/wes/Wes2CromwellInterface.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.{ActorRef}
+import akka.http.scaladsl.model.{HttpHeader}
+import akka.util.Timeout
+import com.typesafe.config.ConfigFactory
+import net.ceedubs.ficus.Ficus._
+import cromwell.services.metadata.MetadataService
+import cromwell.webservice.routes.MetadataRouteSupport.metadataQueryRequest
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.{ExecutionContext, Future}
+
+final class Wes2CromwellInterface()(implicit ec: ExecutionContext) {
```

Comment:
TOL: does this need to be a separate class? Looks like you could probably move the functions inside the `WesRunRoutes` class and save yourself from having to instantiate separately. Alternatively if you like having them in separate files for neatness, you could probably make this a `trait` and mix it in to the `WesRunRoutes` something like 

```scala
trait Wes2CromwellInterface {
  implicit val ec: ExecutionContext
  implicit val duration: FiniteDuration
  implicit val timeout: Timeout
}
```

```scala
trait WesRunRoutes extends Wes2CromwellInterface { ...
```